react/BarButton/Readme.md

```diff
@@ -0,0 +1,48 @@
+# BarButton
+This component is used to display a back Button into the Cozy Bar.
+
+### Example
+
+```jsx static
+  const { BarLeft } = cozy.bar
+  return (
+    <BarLeft>
+      <BarButton />
+    </BarLeft>
+    )
+```
+
+## Props
+List of props causing rendering variations
+
+### `disabled`
+
+Renders the component using a lighter grey color.
+
+```jsx
+<BarButton icon="back" disabled />
+```
+
+### `href`
+
+Renders the component using a anchor tag
+
+```jsx
+<BarButton icon="upload" href="http://cozy.io" />
+```
+
+### `icon`
+
+Used to render the internal `<Icon />` component.
+
+```jsx
+<BarButton icon="cube" />
+```
+
+### `onClick`
+
+Add an `onClick` event handler on the component.
+
+```jsx
+<BarButton icon="gear" onClick={() => alert('BarButton has been clicked')} />
+```
```

Comment:
I think we should only document via Proptypes.

Reply:
Don't you think it's pretty convenient to have those examples rendered in styleguidist to test them manually ?

It allows us to get the component snapshoted by Jest as well.

Reply:
> It allows us to get the component snapshoted by Jest as well.

You're right, I thought that with `jsx` we had only the code and not the component displayed. I forgot that it was `static` that was doing this.